stubs/keyboard/keyboard/mouse.pyi

```diff
@@ -0,0 +1,76 @@
+import sys
+from collections.abc import Callable, Iterable
+from ctypes import c_long
+from typing_extensions import Literal, TypeAlias
+
+from ._generic import GenericListener as _GenericListener
+from ._mouse_event import (
+    DOUBLE as DOUBLE,
+    DOWN as DOWN,
+    LEFT as LEFT,
+    MIDDLE as MIDDLE,
+    RIGHT as RIGHT,
+    UP as UP,
+    X2 as X2,
+    ButtonEvent as ButtonEvent,
+    MoveEvent as MoveEvent,
+    WheelEvent as WheelEvent,
+    X as X,
+    _MouseButton,
+    _MouseEvent,
+    _MouseEventType,
+)
+
+# Can't use ParamSpecArgs on `args`, only on `*args`
+# _P = ParamSpec("_P")
+_P: TypeAlias = tuple[object, ...]
+_Callback: TypeAlias = Callable[[_MouseEvent], bool | None]
+
+class _MouseListener(_GenericListener):
+    def init(self) -> None: ...
+    def pre_process_event(  # type: ignore[override]  # Mouse specific events and return
+        self, event: _MouseEvent
+    ) -> Literal[True]: ...
+    def listen(self) -> None: ...
+
+def is_pressed(button: _MouseButton = ...): ...
+def press(button: _MouseButton = ...) -> None: ...
+def release(button: _MouseButton = ...) -> None: ...
+def click(button: _MouseButton = ...) -> None: ...
+def double_click(button: _MouseButton = ...) -> None: ...
+def right_click() -> None: ...
+def wheel(delta: int = ...) -> None: ...
+def move(x: int | c_long, y: int | c_long, absolute: bool = ..., duration: float = ...) -> None: ...
```

Comment:
```suggestion
def move(x: SupportsInt, y: SupportsInt, absolute: bool = ..., duration: float = ...) -> None: ...
```
It calls `int()` on x and y. Strictly a few more types are allowed: https://github.com/python/typeshed/blob/8e4b89a707f2508e4938e0691b714e235d1719fd/stdlib/builtins.pyi#L200

Reply:
Didn't know about that one. Thanks!